Update README.md (#47)
vtex-apps/shipping-calculator: docs/README.md

> 📢 Use this project, [contribute](https://github.com/{OrganizationName}/{AppName}) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).

Reviewer: ```suggestion
📢 Use this project, [contribute](https://github.com/vtex-apps/shipping-calculator) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
```

> # SHIPPING CALCULATOR

Reviewer: ```suggestion
# Shipping Calculator
```

> ⚠️ **This is an ongoing, unsupported, unfinished and undocumented project. We do not guarantee any results after installation.** ⚠️

Reviewer: ```suggestion
> :warning: *Although ready to be installed, the Shipping Calculator app is currently an **open-beta project**. Bear in mind that due to that status, you can expect an accelerated state of evolution.*
```

<!-- DOCS-IGNORE:start -->
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
<!-- DOCS-IGNORE:end -->

> TBC

Reviewer: It would be good if we added a description regarding the app functionality :)

## Configuration 

> TBC

Reviewer: ```suggestion
🚧 To be defined.
```

## Customization

> TBC

Reviewer: ```suggestion
🚧 To be defined.
```

<!-- ALL-CONTRIBUTORS-LIST:END -->

This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind are welcome!

<!-- DOCS-IGNORE:end -->

> ---- 
> 
> Check out some documentation models that are already live: 
> - [Breadcrumb](https://github.com/vtex-apps/breadcrumb)
> - [Image](https://vtex.io/docs/components/general/vtex.store-components/image)
> - [Condition Layout](https://vtex.io/docs/components/all/vtex.condition-layout@1.1.6/)
> - [Add To Cart Button](https://vtex.io/docs/components/content-blocks/vtex.add-to-cart-button@0.9.0/)
> - [Store Form](https://vtex.io/docs/components/all/vtex.store-form@0.3.4/)

Reviewer: ```suggestion
```
